Tasks/BelkevichAleksandr/Task1/task1.py

```diff
@@ -1,14 +1,13 @@
-a = 3
-b = -14
-c = -5
+deposit = int(input("Input your deposit: "))
+tern = int(input("Input deposit terns: ")) * 12
+persentage = int(input("Input annual percentage: ")) / 100
 
-disc = b**2 - (4 * a * c)
+count_mouth = 0
 
-if disc < 0:
-    print("Корней нет")
-elif disc == 0:
-    print(-(b/2*a))
-elif disc > 0:
-    print((-b + disc*0.5) / 2*a)
-    print((-b + disc*0.5) / 2*a)
+while count_mouth < tern:
+    month_capital = deposit * (persentage) / 12
+    deposit += month_capital
+    count_mouth += 1
+    print(f"Interest on deposit for {count_mouth} mouth = {month_capital}")
```

Comment:
OK, so there is a specific formula to calculate such deposits, and it works better than the cycle which is not a necessary part of the solution, so I will slice 1 point from the score

but it's good that you know how to work with it